docs/src/use/migrate-to-9.0.0.md

```diff
@@ -33,6 +33,7 @@ The lists below are ordered roughly by the number of users each change is expect
 * [`no-restricted-imports` now accepts multiple config entries with the same `name`](#no-restricted-imports)
 * [`"eslint:recommended"` and `"eslint:all"` strings no longer accepted in flat config](#string-config)
 * [`no-inner-declarations` has a new default behavior with a new option](#no-inner-declarations)
+* [`no-unused-vars` now defaults `caughtErrors` to `"all"`](#no-unused-vars)
```

Comment:
As an aside, it would be nice in a separate PR to group all default rule option changes together in this list or a sub-list.